modules/nw-metallb-configure-l2-advertisement-interface.adoc

```diff
@@ -0,0 +1,74 @@
+// Module included in the following assemblies:
+//
+// * networking/metallb/about-advertising-ipaddresspool.adoc
+
+:_content-type: PROCEDURE
+[id="nw-metallb-configure-with-L2-advertisement-interface_{context}"]
+= Configuring MetalLB with a L2 advertisement with interface
+
+By default, the `IPAddressPool` is advertised from all the network interfaces. The `interfaces` field in the `L2Advertisement` custom resource definitions is used to restrict the network interfaces from which the `IPAddressPool` is advertised.
```

Comment:
By default the IP Address from 'IPAddressPool' that has been assigned to the service is advertised from all the network interfaces.

Reply:
@asood-rh I have incorporated the suggested wording. Can you review and let me know if it looks good now?